packages/editor/src/client/routes/articleList.tsx

```diff
@@ -355,7 +353,7 @@ export function ArticleList() {
                   break
 
                 case ConfirmAction.Duplicate:
-                  output = await duplicateArticle({
+                  await duplicateArticle({
```

Comment:
i think you don't need the await if you have the structure `duplicateArticle.then()`. The await is needed when you want it in the variable like we did it before.